core/utils/password.go

```diff
@@ -1,78 +1,28 @@
 package utils
 
 import (
-	"errors"
 	"fmt"
-	"regexp"
-	"unicode"
 
+	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 )
 
 const PasswordComplexityRequirements = `
-Must be longer than 12 characters
-Must comprise at least 3 of:
-	lowercase characters
-	uppercase characters
-	numbers
-	symbols
+Must be longer than 16 characters
 Must not comprise:
 	A user's API email
```

Comment:
Do you want to keep "A user's API email"?

Reply:
Yes, this applies to user passwords